### PR TITLE
Rel 5 0 ut restore database part28

### PR DESCRIPTION
**Pull request by s7design**
Hi @mgruner ,

- Updated Unit tests

Regards,
Zoran

### CODE REVIEW

File: scripts/test/Daemon/DaemonModules/SchedulerTaskWorker/AsynchronousExecutor.t [hunk 1]
```diff
@@ -179,7 +185,7 @@ my @Tests = (
 
 # get task handler objects
 my $TaskHandlerObject
-    = $Kernel::OM->Get('Kernel::System::Daemon::DaemonModules::SchedulerTaskWorker::AsynchronousExecutor');
+    = $Kernel::OM->Get('Kernel::System::Daemon::DaemonModules::SchedulerTaskWorker::AsynchronousExecutorF');
```
Comment (mgruner) [inline scripts/test/Daemon/DaemonModules/SchedulerTaskWorker/AsynchronousExecutor.t]: There seems to be a typo here.

**Comment by mgruner** [comment]
I also get warnings about "rollback ineffective with AutoCommit enabled". Could it be that there are new DBObjects created? In that case RestoreDatabase would not work.

**Comment by s7design** [comment]
Hi @mgruner 

Thanks, yes it was my mistake. I updated PR

Regards
Zoran

**Comment by mgruner** [comment]
Merged, thanks! I also added a check to the Helper to cause a test failure if the rollback fails to make sure this does not happen in other test cases as well.

**Pull request closed without merging**